Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -189,6 +241,36 @@ public bool TryGetNearGraspPoint(out Vector3 result)
             }
         }
 
+
+        private static readonly ProfilerMarker TryGetNearGraspAxisPerfMarker = new ProfilerMarker("[MRTK] ConePointer.TryGetNearGraspAxis");
+
+        /// <summary>
+        /// Gets the axis that the grasp happens
+        /// For the SpherePointer it's the axis from the palm to the index tip
+        /// For any other IMixedRealityController, return just the pointer's forward orientation
+        /// </summary>
+        public bool TryGetNearGraspAxis(out Vector3 result)
```

Comment:
The parameter is named `axis` in the interface, so it might be good to leave it consistent here.